scripts/llnl_scripts/batch_scripts/README.md

```diff
@@ -0,0 +1,26 @@
+[comment]: # (#################################################################)
```

Comment:
Since the `Makefile` for the batch scripts was fundamentally broken, I converted it to a `README`